### PR TITLE
Add: solid and outlined buttons

### PR DESCRIPTION
**Pull request by guacamole**
Signed-off-by: guacamole <gunjanwalecha@gmail.com>

### CODE REVIEW

File: src/lib/button-outlined.svelte [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+	// export let label = '';
```
Comment (jay-dee7) [inline src/lib/button-outlined.svelte:2]: @guacamole do we need this one? if not let's remove it

**Review by jay-dee7 — CHANGES_REQUESTED** [review]
@guacamole tiny change and we're good

File: src/lib/button-outlined.svelte (hunk 1 above)
Comment (guacamole) [reply src/lib/button-outlined.svelte]: you got it!

**Review by jay-dee7 — APPROVED** [review]
LGTM @guacamole 🥇 🥇 🥇

**Pull request merged**